src/components/Card/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Card';
```

Comment:
also reexport from `src/components/index.ts` is required

Reply:
912644e